src/WalkingModule/app/robots/ergoCubSN002/dcm_walking_with_joypad.ini

```diff
@@ -0,0 +1,75 @@
+# Remove this line if you don't want to use the MPC
+# use_mpc                            1
+
+# Remove this line if you don't want to use the QP-IK
+use_QP-IK                          1
+
+# remove this line if you don't want to save data of the experiment
+dump_data                          1
+
+# Limit on the maximum initial velocity. This avoids the robot to jump at startup
+max_initial_com_vel                 0.15
+
+# If set to true, the desired ZMP is used directly in the COM/ZMP controller
+skip_dcm_controller                  1
+
+# The goal port receives 2 or 3 doubles according to the selected planner controller
+goal_port_suffix                    /goal:i
+
+# Scales the input coming from the goal port
+goal_port_scaling                   (0.5, 1.0, 0.5)
+
+# How much in advance the planner should be called. The time is in seconds
+planner_advance_time_in_s           0.08
```

Comment:
This value is other than the others. Also the ``max_feedback_delay_in_s`` is missing. I guess we had this issue also in the other ergoCub

Reply:
Done here: https://github.com/robotology/walking-controllers/pull/175/commits/be143cb5f6880e35264a4e8877cd08927df4768d